qa/rpc-tests/cfund-paymentrequest-extract-funds.py

```diff
@@ -47,15 +47,21 @@ def run_test(self):
 
         # Create 6 payment requests
```

Comment:
I think this should be `5` instead of `6`

Reply:
it creates 6, 5 correct and the 6th should be rejected